desktop/config-updater.json

```
@@ -1,8 +1,8 @@
 {
-  "forceUpdate": false,
   "updater": {
-    "url":
-      "https://app.simplenote.com/desktop/{platform}/version?compare={version}",
+    "downloadUrl": "https://github.com/Automattic/simplenote-electron/releases/latest",
+    "changelogUrl": "https://github.com/Automattic/simplenote-electron/blob/master/changelog.md",
```

```suggestion
    "changelogUrl": "https://github.com/Automattic/simplenote-electron/blob/master/CHANGELOG.md",
```

Just a small tweak to the file name casing.